Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.MacOS/AppDelegate.cs

```diff
@@ -4,7 +4,10 @@
 using AppKit;
 using Foundation;
 using Xamarin.Forms.Platform.MacOS;
+using Xamarin.Forms;
```

Comment:
We do we need these usings?

Reply:
using Xamarin.Forms for 'Dependency'
using Contoso.Forms.Demo.MacOS; for 'AppDelegate'